client/internal/url/url.go

```diff
@@ -149,3 +149,11 @@ func ReadFmcTaskStatus(baseUrl string, fmcDomainUid string, taskId string) strin
 func ReadTenantDetails(baseUrl string) string {
 	return fmt.Sprintf("%s/anubis/rest/v1/oauth/check_token", baseUrl)
 }
+
+func CreateApplication(baseUrl string) string {
+	return fmt.Sprintf("%s/aegis/rest/v1/services/targets/applications", baseUrl)
+}
+
+func ReadApplication(baseUrl string) string {
+	return fmt.Sprintf("%s/aegis/rest/v1/services/targets/applications", baseUrl)
```

Comment:
I would remove that since it's duplicated and call it `CreateOrReadApplication` instead

Reply:
Yea agree, we should perform some refactoring to remove duplicates in this file in general, but I would keep that as a separate PR because I do not want to leave things as partially fixed